README.md

```diff
@@ -104,6 +115,10 @@ vars:
     snapchat_ads__campaign_hourly_report_passthrough_metrics:
       - name: "unique_string_field"
         alias: "field_id"
+    snapchat_ads__campaign_daily_country_report_passthrough_metrics:
+      - name: "new_measure_country_report"
+    snapchat_ads__campaign_daily_region_report_passthrough_metrics:
+      - name: "new_measure_region_report"
```

Comment:
Could you note the end models these persist to (and the source tables they come from)